src/command-registry.coffee

```diff
@@ -18,6 +18,11 @@ SequenceCount = 0
 # command event listeners globally on `atom.commands` and constrain them to
 # specific kinds of elements with CSS selectors.
 #
+# Command names must follow the `namespace:event` pattern, where both parts can
+# be further split into words by hyphens. Typically `namespace` will be the
```

Comment:
I would go ahead and make the same **strong** recommendation that `namespace` *must* be the dasherized version of your package name.